contrib/internal/httptrace/httptrace.go

```diff
@@ -156,3 +154,37 @@ func isGlobal(ip netaddr.IP) bool {
 	}
 	return isGlobal
 }
+
+// getURL returns the full URL of the http request. If query params are collected, they will be obfuscated granted
+// obfuscation is not disabled by the user (through DD_TRACE_OBFUSCATION_QUERY_STRING_REGEXP)
+// For more information see https://datadoghq.atlassian.net/wiki/spaces/APM/pages/2357395856/Span+attributes#http.url
+func getURL(r *http.Request) string {
```

Comment:
Would it be OK to call this `requestURL` or `urlFromRequest`?
```suggestion
func requestURL(r *http.Request) string {
```

Reply:
I'm in favour of `urlFromRequest`, `requestURL` may be ambiguous.